test/app-for-e2e/pkg/handlers/handlers.go

```diff
@@ -112,16 +91,117 @@ func NewMemLoadHandler(res http.ResponseWriter, req *http.Request) {
 	log.Println("Leaving NewMemLoadHandler")
 }
 
-// ListRoutesHandler returns the valid paths on this server.
+// ListRoutesHandler returns available paths
 func ListRoutesHandler(res http.ResponseWriter, req *http.Request) {
-	log.Println("Entered ListRoutesHandler")
-	paths := []string{"cpu", "mem"}
-	for _, path := range paths {
-		fmt.Fprintln(res, path)
+	paths := "paths: / , cpu , newmem , pet , pupulate"
+	messageResponseJSON(res, http.StatusOK, paths)
+	return
+}
+
+// DEMO APP
+
+// MessageResponse contains any additional information required in the server response
+type MessageResponse struct {
+	Message string `json:"message"`
+}
+
+// PetHandler handles pets and pet accessories
+// ex: ex: localhost:1738/pet , ex: localhost:1738/pet?petId=4
+func PetHandler(res http.ResponseWriter, req *http.Request) {
+	petIdParam, _ := req.URL.Query()["petId"]
+
+	//GET
```

Comment:
Wanted to get this out "quickly", but should use a router

Reply:
no worries, a quick fix until you can use a router to clean it up a bit is to make them their own functions and then a switch on req.Method.